### PR TITLE
Editpage

### PR DESCRIPTION
**Pull request by Mernawaseif**
@Veegas @MinaNemr  @KarimElSheikh @Israa-Elwan  @nelsammak  
could you please review.

### CODE REVIEW

**Comment by Veegas** [comment]
@Mernawaseif  Sheely bas file profile.js~ dah maloosh lazma

File: backend/views/profile.js [hunk 1]
```diff
@@ -101,81 +101,34 @@ exports.deletePhoto = function(profilePhoto) {
 
  
 
-  // PUT
-   var editProfile = function (req, res, next) {
+  /*
```
Comment (MinaNemr) [inline backend/views/profile.js]: You need to change the function documentation to be auto-generated afterwards
example
/**
    \*   @function getUsers - Called on GET "/api/users" 
    \*   @params {Object} req - Http request
    \* @params {Object} res - Http response
    \*   @params {Object} next - Next middleware
    \*   @returns {JSON} All users in Database as JSON
    */

File: backend/views/profile.js~ [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+  fs.unlink(photoPath + profilePhoto, function(err) {
+    if (err) {
+      console.log(err);
+      return;
+    }
+
+    console.log('Successfully delete the profile Photo');
+  });
+};
+
```
Comment (MinaNemr) [inline backend/views/profile.js~:90]: @Mernawaseif document

File: backend/views/profile.js~ [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+       return next(err);
+      } else {
+        console.log(profile);
+        res.json({profile: profile});
+      }
+    });
+  };
+
+ 
+
```
Comment (MinaNemr) [inline backend/views/profile.js~:105]: @Mernawaseif document

File: frontend/scripts/controllers/editctrl.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+angular.module('angularPassportApp')
+  .controller('EditCtrl', function ($scope,$http,$location) {
+   /**
+  * function edit 
+  * take info of user from edit form and post it to the backend
+  */
```
Comment (MinaNemr) [inline frontend/scripts/controllers/editctrl.js]: @Mernawaseif please document function like the example i gave u

File: frontend/views/partials/editpage.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
+<div class="container">
```
Comment (MinaNemr) [inline frontend/views/partials/editpage.html:1]: @Mernawaseif 
please add section identifier to know what begins where
like
<!-edit form starts here-->  
 Html code
<!-edit form ends here-->

**Comment by Mernawaseif** [comment]
@MinaNemr pls review again the documentation

File: backend/views/profile.js [hunk 6]
```diff
@@ -101,81 +101,36 @@ exports.deletePhoto = function(profilePhoto) {
 
  
 
-  // PUT
-   var editProfile = function (req, res, next) {
+/**
```
Comment (MinaNemr) [inline backend/views/profile.js:105]: @param instead of @ params
mention param after description
sorry i told u it is params before

**Comment by Israa-Elwan** [comment]
@Mernawaseif  scenario reviewed

**Comment by KarimElSheikh** [comment]
@Mernawaseif I didn't find the test files

**Pull request merged**